### ISSUE TITLE
Error related to virtual environment

### ISSUE THREAD

**Issue by maxawake** [Sharktheone/arch-mojo#3]
Hey, first of all thanks for creating this script! Its really a pity the developers of mojo don't distribute platform independent.

I wanted to use your script to install mojo and issues the command 
```
mkdir -p /tmp/arch-mojo && curl -sSL https://raw.githubusercontent.com/Sharktheone/arch-mojo/main/install.py -o /tmp/arch-mojo/install.py && python /tmp/arch-mojo/install.py
Do you have modular already installed? (y/n): n
Do you want to install the libraries globally (for all users)? (y/n): y
Do you want to automatically use a venv when running modular install/update? (y/n): y
Where do you want to create the venv?: ~/Apps/arch-mojo

```

which lead to the following error

```
modular: error: failed to run python: Executable "$_self.package_path/venv/bin/python" doesn't exist!
```

Has anybody faced a similar issue and maybe even got a solution at hand?

**Comment by maxawake** [comment]
I should add that i am able to run mojo using 
```
source ~/.modular/pkg/packages.modular.com_mojo/venv/bin/activate 
mojo
```

**Comment by Sharktheone** [comment]
I also noticed this when i first tried the new Mojo release, but it worked for me. Since i am currently working on a updated version of the script, so it should theoretically work with the AUR, so I didn't update the script to fix this.

**Comment by Sharktheone** [comment]
I think, I can close this now, since I have fixed this with de2291b2a1c5f66d123e587c5f90a01b3db60010
